ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -66,6 +66,24 @@ SettingsContentBase {
             Layout.fillWidth: true
             currentIndex: editPreviwTabBar.currentIndex
 
+            onImplicitHeightChanged: {
+                // During opening Profile, StackLayout always set implicitHeight of his biggest
+                // child implicitHeight and it reflecting on a scroll area.
+                // Use implicitHeight of current child
+                if (currentIndex === 0 && implicitHeight !== settingsView.implicitHeight) {
+                    implicitHeight = settingsView.implicitHeight
+                } else if (currentIndex === 1 && implicitHeight !== profilePreview.implicitHeight) {
+                    implicitHeight = profilePreview.implicitHeight
+                }
+            }
```

Comment:
In general, I don't like this imperative approach; also hardcoding "currentIndex" to some magic values

Reply:
Good idea to give some names for this values :)

Reply:
fixed